frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -79,17 +79,14 @@ export const TextRow = ({
             confirmText={t('schema_editor.textRow-deletion-confirm')}
             onConfirm={handleDeleteClick}
             onClose={() => setIsConfirmDeleteDialogOpen(false)}
-            trigger={
-              <StudioButton
-                className={classes.deleteButton}
-                icon={<TrashIcon title={`Slett ${textId}`} />}
-                variant='tertiary'
-                onClick={() => setIsConfirmDeleteDialogOpen((prevState) => !prevState)}
-                aria-label={t('schema_editor.delete')}
-              >
-                {t('schema_editor.delete')}
-              </StudioButton>
-            }
+            triggerProps={{
+              className: classes.deleteButton,
+              icon: <TrashIcon title={`Slett ${textId}`} />,
```

Comment:
I understand that this has nothing to do with your changes, but I see we have a text key; `"general.delete_item": "Slett {{item}}"` that we could replace the hardcoded title with 🙈